brighthive/data-resource-api
# Feat/delete many to many

## Overview

This adds the ability to perform DELETE operations to the many to many relationships.

### Checklist


## Testing Instructions

1. Test many to many DELETE

 * Start after checking out this branch
 * Build the image with `docker build -t brighthive/data-resource-api:1.0.0-alpha .`
 * Run `docker-compose up`
 * Give the application a minute or two to spin up
 * `POST` four credentials to `http://localhost:8000/credentials`
```
{
	"credential_name": "credential test"
}
```

 * `POST` a program with `'credentials': [1,2,3,4]` to `http://localhost:8000/programs`
```
{
	"program_name": "aprogram 1",
	"program_code": 1,
	"program_description": ".",
	"program_status": ".",
	"program_fees": 1,
	"eligibility_criteria": ".",
	"program_url": ".",
	"credentials": [1,2,3,4]
}
```

 * `DELETE` the following to `http://localhost:8000/programs/1/credentials`. It should return `'credentials': [2,3,4]`
```
{
	"credentials": [1]
}
```

 * `DELETE` the following to `http://localhost:8000/programs/1/credentials`. It should return `'credentials': [4]`
```
{
	"credentials": [5,2,3]
}
```
